Training/Scenes/Main/Subviews/EventTableViewCell.swift

```diff
@@ -1,9 +1,19 @@
 import UIKit
+import Nuke
 
 final class EventTableViewCell: UITableViewCell {
     func event(_ event: Event) {
-        label.text = "\(event.type) by \(event.actor.login) at \(event.repo.name)"
+        label.text = event.type
+        descriptionLabel.text = event.repo.name
+        Nuke.loadImage(with: event.actor.avatarUrl, into: profileImage)
+    }
+    
+    @IBOutlet var profileImage: UIImageView!{
```

Comment:
[nits]
基本的に外からアクセスされない or アクセスされたくないものには `private` などのアクセス修飾子を付けたほうが良いです！
メリットは以下の2つです！
- コードを読むときに`profileImage`がアクセスされる(変更される)箇所が明確になる→コードを把握しやすくなる
- ビルドの時間がかなり少しですが減る

https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html

```suggestion
    @IBOutlet private var profileImage: UIImageView!{
```